monitor.go

```diff
@@ -75,8 +81,8 @@ func newStageMonitor(g reflect.Value, v specsp) (*stageMonitor, error) {
 	x := v["x"].(float64)
```

Comment:
In the current implementation, for widgets' position, `x: 0, y: 0` means the **left-top corner** (of viewport); while for sprites' position, `x: 0, y: 0` means the **center point** (of map).

Though the positioning behavior of widgets and sprites is different: widgets are positioned relative to the viewport, and sprites are positioned relative to the map. I believe it'll be better to keep the convention that `x: 0, y: 0` means the **center point** for both.

@JiepengTan @xushiwei What do you think?

Reply:
And the direction of the coordinate system. Currently, for widgets, the Y-axis points down, while for sprites, the Y-axis points up; I suggest we unify this as well.